resources/pt/cemu.po

```diff
@@ -849,20 +827,21 @@ msgid ""
 "\n"
 "Recommended: Auto"
 msgstr ""
-"Shaders pré-compilados podem acelerar o tempo de carregamento dos shaders no "
-"início da emulação.\n"
-"A opção \"auto\" ativará isto para AMD/Intel, mas deixará desativado em GPUs "
-"da NVIDIA como solução para um bug no driver.\n"
+"Shaders pré-compilados podem acelerar o tempo de\n"
+"carregamento dos shaders no início da emulação.\n"
+"A opção \"Automático\" ativará isto para AMD/Intel,\n"
+"mas deixará desativado em GPUs da NVIDIA\n"
+"como solução para um bug no driver.\n"
```

Comment:
One last thing! You forgot to revert line breaks in this string. Everything else is fine!